Makefile

```diff
@@ -88,10 +88,10 @@ release: clean qa readme authors sdist doc
 	@mkdir release
 	@cp dist/*.gz release/
 	@cp cobbler.spec release/
-	@cp debian/cobbler.dsc release/
```

Comment:
Would you mind removing them. We removed the debian packaging folder completly as we are doing a cross platform build in the OBS currently.